app/src/main/java/ch/epfl/sdp/appart/user/User.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sdp.appart.user;
+
+public interface User {
+
+    /**
+     * @return the name of the user
+     */
+    public String getName();
+
+    /**
+     * sets the name of the user
+     */
+    public void setName();
+
+    /**
+     * getter method for ID
+     * @return the unique id of the user
+     */
+     public String getUserId();
+
+    /**
+     * setter method for user email
+     */
+    public void setUserEmail();
+
+    /**
+     * getter method for user email
+     * @return user email as a string
+     */
+     public String getUserEmail();
+
+    /**
+     * setter method for user phone number
+     */
+    public void setPhoneNumber();
+
+    /**
+     * getter method for user phone number
+     * @return user phone number as a string
+     */
+     public String getPhoneNumber();
+
+    /**
+     * setter method for user profile picture
+     */
+     public void setProfileImage();
+
+    /**
+     * getter method for user profile picture
+     * @return the profile picture (Object type was chosen as generic, should be changed once the object Picture is identified)
+     */
+    public Object getProfileImage();
```

Comment:
There is an [Image](https://developer.android.com/reference/android/media/Image) class that extends Object. I think we can use this type without losing generality.